mllib/src/test/scala/org/apache/spark/ml/source/libsvm/LibSVMRelationSuite.scala

```diff
@@ -109,14 +112,15 @@ class LibSVMRelationSuite extends SparkFunSuite with MLlibTestSparkContext {
   test("write libsvm data and read it again") {
     val df = spark.read.format("libsvm").load(path)
     val tempDir2 = new File(tempDir, "read_write_test")
```

Comment:
`Utils.createTempDir` seems to be a nicer way. The directory is automatically deleted when VM shuts down, so I believe no manual cleanup (cf. comment lower) is needed.